File: C#/Demo/DemoConsol/Models/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DemoConsol.Models
{
    
    /// <summary>
    /// provide demo of partial class
    /// </summary>
    public partial class Account
    {
        public string AccountHolder { get; set; }
        public decimal Balance { get; set; }

    }


    /// <summary>
    /// second part of the demo of partial class 
    /// </summary>
    public partial class Account
    {
        public void Deposit(decimal amount)
        {
            Balance += amount;
            Console.WriteLine($"{amount} deposited. New Balance: {Balance}");
        }

        public void Withdraw(decimal amount)
        {
            if (amount > Balance)
            {
                Console.WriteLine("Insufficient balance!");
                return;
            }
            Balance -= amount;
            Console.WriteLine($"{amount} withdrawn. New Balance: {Balance}");
        }
    }

}

File: JavaScript/Assignment/7_Module_Split/Account.js
class BankAccount{
    accountHolder;
    #balance;

    constructor(name,amount){
        this.accountHolder = name;
        this.#balance=amount;
    }

    deposite(amount){
        this.#balance += amount;
    }

    withdraw(amount){
        this.#balance -= amount;
    }

    get balance(){
        return this.#balance;
    }

    static info(){
        return "Bank System v1.0";
    }
}

export default BankAccount;